index.html

```diff
@@ -630,7 +630,43 @@ <h2>DID Documents</h2>
     </section>
 
     <section>
-      <h2>DID Resolvers and DID Resolution</h2>
+      <h2>DID Resolvers</h2>
+
+      <p>
+A <a>DID resolver</a> is a software and/or hardware component that takes a
```

Comment:
It's confusing to talk about the component first and then the process that defines them, I would suggest that this be reversed. However, realizing that this section is defining components of the architecture and not actions of the architecture, this ordering might be desired. With that in mind, perhaps these pieces should be moved to the resolution section below instead.

Reply:
As discussed on the call, leaving this as-is for now to talk about the "nouns" in the architecture, and then what the nouns do.